PresentMon/CsvOutput.cpp

```diff
@@ -232,6 +238,17 @@ void WriteCsvRow<FrameMetrics1>(
         fwprintf(fp, L",%.*lf", DBL_DIG - 1, 0.001 * pmSession.TimestampDeltaToMilliSeconds(p.PresentStartTime));
         break;
     }
+    if (args.mWriteDisplayTime) {
```

Comment:
Two things on this one. 1. I think it's backwards. When ScreenTime is 0 then we want NA. 2. PresentEvent.ScreenTime is a uint64_t so I don't think we should compare with "0.f".

Reply:
Oh do it like below. :)

Reply:
good catch, this is for the old metrics format so I catch that when I checked the running output!